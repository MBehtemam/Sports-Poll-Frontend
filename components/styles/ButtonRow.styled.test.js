import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import { mount } from "enzyme";
import "jest-styled-components";
import ButtonRowStyled from "./ButtonRow.styled";

describe("Test suits for Button row", () => {
  it("should match with snapshot", () => {
    const tree = renderer.create(<ButtonRowStyled />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
