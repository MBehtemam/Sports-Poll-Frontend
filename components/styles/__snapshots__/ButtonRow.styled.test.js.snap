// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Button row should match with snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  margin-top: 10px;
}

.c0 button {
  border: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 128px;
  height: 64px;
  background-color: #ffe484;
  border-color: #ffe484;
  font-size: 1rem;
  font-weight: 500;
}

<section
  className="c0"
/>
`;
